schemas/json/font-collection.json

```diff
@@ -154,5 +160,5 @@
 		}
 	},
 	"additionalProperties": false,
-	"required": [ "$schema", "slug", "name", "font_families" ]
+	"required": [ "font_families" ]
```

Comment:
Only `font_families` should actually be required. If the `$schema` property is not defined in your json file, this required field itself will not be read, so I think `$schema` is unnecessary.